scanpointgenerator/generators/arraygenerator.py

```diff
@@ -0,0 +1,71 @@
+from scanpointgenerator.compat import np
+from scanpointgenerator.core import Generator
+
+
+@Generator.register_subclass("scanpointgenerator:generator/ArrayGenerator:1.0")
+class ArrayGenerator(Generator):
+    """Generate points fron a given list of positions"""
+
+    def __init__(self, names, units, points, alternate_direction=False):
+        self.names = names
+        self.units = units
+        self.alternate_direction = alternate_direction
+        self.points = np.array(points, dtype=np.float64)
+        if self.points.shape == (len(self.points),):
```

Comment:
ok, but we are reducing the array generator to 1-dim, so that should cut a lot of this code out...